src/components/ConversationSettings/ConversationSettingsDialog.vue

```diff
@@ -24,19 +24,32 @@
 		role="dialog"
 		:aria-label="t('spreed', 'Conversation settings')"
 		:open.sync="showSettings"
-		:show-navigation="false">
+		:show-navigation="true">
```

Comment:
```suggestion
		:show-navigation="canFullModerate">
```

Could only show it for moderators, but I guess it's fine as well like it is currently